data-service/README.md

```diff
@@ -32,18 +71,20 @@ The client request will contain the token and resource id that is used to unique
 |                 |                        | serverMetadata       | error             |
 |                 |                        |                      | serverMetadata    |
   
-*token comes in the body of the request from the client (DataRequest) and is stored in the header metadata for the audit messages
+*token comes in the body of the request from the client (DataRequest) and is stored in the header metadata for the 
+audit messages
 
 **attributes will include the numbers for records processed and records returned
 
-***data that may not be available depending on when the error occurred
+***data types that may not be available depending on when the error occurred
+
 
 ## REST Interface
 
 The application exposes one endpoint to the client for retrieving the resources. This will be the data that has previously 
-been requested and prepared in the initial request to Palisade services. 
+been requested and prepared in the initial request to the Palisade services. 
 * `POST data/read/chunked`
-    - returns a `202 ACCEPTED` and a streamed HTTP response body which will provides the resource.
+    - returns a `200 OK` and a streamed HTTP response body which will provide the resource(s).
```

Comment:
The short of it is that I don't know.  A 202 tells us that they "got" the message, but we don't know what they did with it.  The successful response we do get back will be one with an output stream and no guarantee that everything will work.  What kind of a 200 status is that?

Reply:
Getting into the nitty-gritty - 
https://tools.ietf.org/html/rfc7231#section-6.3.1
Since we always return a body (even if it's a stream), it should be a 200 OK, so no change needed